operations/iuf/examples/iuf_activity.md

```diff
@@ -1,6 +1,6 @@
 # `iuf activity` Examples
 
-(`ncn-m001#`) Display an activity.
+(`ncn-m001#`) Display activity `admin-230126`.
 
 ```bash
 iuf -a admin-230126 activity
```

Comment:
at the moment, you must supply -m also

Reply:
Not for activity.  If this is a bug, let's chat.  Otherwise I can leave it as it is?

```
ncn-m001:/etc/cray/upgrade/csm # iuf -a johnn-230201 activity | head -10
+---------------------------------------------------------------------------------------------------------------------------------------+
| Activity: johnn-230201                                                                                                                |
+---------------------+-------------+--------------------------------------------+-----------+----------+-------------------------------+
| Start               | Category    | Argo Workflow                              | Status    | Duration | Comment                       |
+---------------------+-------------+--------------------------------------------+-----------+----------+-------------------------------+
| 2023-02-01t19:09:56 | in_progress | johnn-230201-qxjal-process-media-8srxc     | Succeeded | 0:04:38  | Run process-media             |
| 2023-02-01t19:14:34 | in_progress | johnn-230201-ksrey-pre-install-check-h74sr | Succeeded | 0:01:07  | Run pre-install-check         |
| 2023-02-01t19:15:41 | in_progress | johnn-230201-ksrey-deliver-product-lh87q   | Failed    | 0:20:31  | Run deliver-product           |
| 2023-02-01t19:36:12 | debug       | None                                       | n/a       | 0:42:11  | None                          |
| 2023-02-01t20:18:23 | in_progress | johnn-230201-633xi-process-media-hd6g2     | Succeeded | 0:03:39  | Run process-media             |
```

Reply:
we might have fixed the bug.  it's preferable not to require -m for activity (and in reality, perhaps anything that doesn't involved process-media)